README.md

```diff
@@ -91,9 +91,10 @@ resource JAR is included on the classpath of Presto coordinator, it will be able
 resources.
 
 None of the Java code relies on the Presto UI project being compiled, so it is possible to exclude
-this UI when building Presto. It can be excluded by disabling the `ui` maven profile with `-P \!ui`:
+this UI when building Presto. Add the property `-DskipUI` to the maven command to disable building
+the `ui` maven module.
 
-    ./mvnw clean install -P \!ui
+    ./mvnw clean install -DskipUI
 
 You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to build the UI. When using  Maven to build
```

Comment:
Is this still true?

Reply:
For development, I believe it's still easier to have them installed on your local system. Technically don't need them since we use the maven plugin now, but having to reference the executable from the `./target` is a little cumbersome. @yhwang may want to update this? It is adjacent to this change though.